_posts/2018-12-29-dask-on-hpc.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title: Dask on HPC
+tagline: what works well, and what needs work
+author: Dharhas Pothina (Army ERDC), Guillaume Eynard-Bontemps (CNES), Kevin Paul (NCAR), Matthew Rocklin (NVIDIA), Willi Rath (GEOMAR), Joe Hamman (NCAR)
+tags: [Programming, Python, scipy, dask]
+draft: true
+theme: twitter
+---
+{% include JB/setup %}
+
+We analyze large datasets on HPC systems with Dask,
```

Comment:
large is a bit vague, can we be more quantitative: 1TB to 100TB?

Reply:
What I've seen is from 100s GB to 100TB, I'm OK if you want to precise it.